tests/test_transforms/test_feature_selection/test_filter_transform.py

```diff
@@ -97,3 +118,70 @@ def test_exclude_filter_wrong_column(ts_with_features):
     transform = FilterFeaturesTransform(exclude=["non-existent-column"])
     with pytest.raises(ValueError, match="Features {.*} are not present in the dataset"):
         ts_with_features.fit_transform([transform])
+
+
+@pytest.mark.parametrize("type", ("include", "exclude"))
+@pytest.mark.parametrize(
+    "columns, expected_excluded_columns, expected_included_columns",
+    [
+        ([], ["target", "exog_1", "exog_2"], []),
+        (["target"], ["exog_1", "exog_2"], ["target"]),
+        (["exog_1", "exog_2"], ["target"], ["exog_1", "exog_2"]),
+        (["target", "exog_1", "exog_2"], [], ["target", "exog_1", "exog_2"]),
+    ],
+)
+def test_inverse_transform_save_columns(
```

Comment:
Let's divide it for the two separate tests for include/exclude type. The logic is slightly different in this cases and it is much easier to read the test without the additional logic included with "ifs"

Reply:
Also this test should cover the cases when `return_features=False` to make sure that the old logic has not been broken

Reply:
It is better to name the test "test_transform_save_columns" because the test is about "transform" but not "inverse_transform"

Reply:
I think it's strange to add case `return_features=False`, because this test checks that transform correctly saved deleted columns, but transform with `return_features=False` don't save anything